### DISCUSSION TITLE
Big progress in july / august

### DISCUSSION THREAD

**Discussion by Psychokrameur** [una-ui/una-ui#172] [General]
Hello, 

I'm following this projet since aprill and there is a big progress the last couple of month!
Just want to highligh this and say thank you to @phojie and @mnenie 

I'm not able to use it and contribute for now due to lack of time but hope to be able to in the next months!

**Comment by phojie** [comment]
Thanks, @Psychokrameur for your support! The core team are always happy to contribute and make progress on this project. 🎊 We really hope we can ship `v1` soon 🚀. Really looking forward to it, @Psychokrameur.

**Comment by mnenie** [comment]
Thanks @Psychokrameur 🤝
